### PR TITLE
add moving background

### PR DESCRIPTION
**Pull request by aswanthkoleri**

### CODE REVIEW

**Comment by aswanthkoleri** [comment]
[Live link](https://aswanthkoleri.github.io/humblefoolcup/)

File: css/style.css [hunk 1]
```diff
@@ -763,3 +763,41 @@ html {
   margin : 10px;
   align-content: center;
 }
+body{
+	margin:0;
+	padding:0;
+	font-family:"arial",heletica,sans-serif;
+	font-size:12px;
+    background: #2980b9 url('http://static.tumblr.com/03fbbc566b081016810402488936fbae/pqpk3dn/MRSmlzpj3/tumblr_static_bg3.png') repeat 0 0;
+	-webkit-animation: 10s linear 0s normal none infinite animate;
```
Comment (shubham-padia) [inline css/style.css]: Please download the image, add it to `humblefool/img/` folder, and change the link here accordingly, rest all looks good :)

**Review by shubham-padia — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 1 above)
Comment (aswanthkoleri) [reply css/style.css]: ok

File: css/style.css [hunk 2]
```diff
@@ -763,3 +763,41 @@ html {
   margin : 10px;
   align-content: center;
 }
+body{
+	margin:0;
+	padding:0;
+	font-family:"arial",heletica,sans-serif;
+	font-size:12px;
+    background: #2980b9 url('humblefool/img/background.png') repeat 0 0;
```
Comment (shubham-padia) [inline css/style.css]: url will be `../humblefool/img/background.png` as humblefool directory is not in the same as the one your file currently iis

**Review by shubham-padia — CHANGES_REQUESTED** [review]

**Comment by pradeepgangwar** [comment]
This moving background is same to what i added while making humblefool tribute page. But @magician03 requested to include the background that is currently on humblefool's page.

It will be nice if @magician03 can confirm. 
Or tell me if i am confusing this change with any other. :-)

**Comment by pradeepgangwar** [comment]
Sorry this change is for Main page i suppose...... I thought it to be for humblefool tribute page. Sorry for misunderstanding so no review needed @shubham-padia sir from @magician03 .  
Its totally ok.

File: css/style.css (hunk 2 above)
Comment (shubham-padia) [reply css/style.css]: Change this please

**Comment by aswanthkoleri** [comment]
fixed it

**Comment by shubham-padia** [comment]
Great squash the commits and youre good to go

**Review by shubham-padia — APPROVED** [review]

**Pull request merged**